pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/UniformLoadShedder.java

```diff
@@ -76,7 +76,7 @@ public Multimap<String, String> findBundlesForUnloading(final LoadData loadData,
         MutableDouble minThroughputRate = new MutableDouble(Integer.MAX_VALUE);
         brokersData.forEach((broker, data) -> {
             //broker with one bundle can't be considered for bundle unloading
-            if (data.getLocalData().getBundles().size() <= 1) {
+            if (data.getLocalData().getBundles().size() == 1) {
```

Comment:
It just selects bundles for unloading. If one broker with one or doesn't have any bundles, it will skip this cycle.

Reply:
It is not only to choose overloaded brokers to unload, but also to choose low-loaded brokers to load.
Otherwise, a new broker cannot be assigned to the bundle

Reply:
Fixed. PTAL,thanks! @hangc0276